extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnChunkReaderImpl.java

```diff
@@ -267,9 +267,7 @@ public ColumnPageReader next(@NotNull final SeekableChannelContext channelContex
                 final long dataOffset = ch.position();
                 nextHeaderOffset = dataOffset + pageHeader.getCompressed_page_size();
                 final PageType pageType = pageHeader.type;
-                if (pageType == PageType.DICTIONARY_PAGE && headerOffset == columnChunk.meta_data.getData_page_offset()
-                        && columnChunk.meta_data.getDictionary_page_offset() == 0) {
-                    // https://stackoverflow.com/questions/55225108/why-is-dictionary-page-offset-0-for-plain-dictionary-encoding
+                if (pageType == PageType.DICTIONARY_PAGE) {
```

Comment:
Note that in line 245, we start looking at data pages from `columnChunk.meta_data.getData_page_offset()`.
Now, for the mentioned file, both `columnChunk.meta_data.getData_page_offset()` and `columnChunk.meta_data.getDictionary_page_offset()` were equal and pointed to the offset of dictionary page.
That is why the check was failing.